app/views/notifications/_form.html.erb

```diff
@@ -2,10 +2,11 @@
   <h3><%= "#{@event.title} [#{@event.id_code}] "  %></h3>
   <h4>(<%= format_time_span(@event.start_time, @event.end_time, @event.duration) %>)</h4>
   <%= f.error_notification %>
+  <div class="alert alert-danger"><%= f.error :base %></div>
```

Comment:
I _think_ if we do what I mentioned re: `errors.add`, we might not need this. Also, does `error_notification` have the same CSS clases?